src/main/java/ru/skypro/homework/dto/CommentDTO.java

```diff
@@ -0,0 +1,22 @@
+package ru.skypro.homework.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.Instant;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+@Builder
+public class CommentDTO {
+    private Integer author;
+    private String authorImage;
+    private String authorFirstName;
+    private Instant createdAt;
```

Comment:
```suggestion
    private Long createdAt;
```
или нужно указать, что при принятии long надо десериализовать это в Instant (по умолчанию вроде бы работать не будет - точно не помню, но проверьте)) )